instrumentation/rack/lib/opentelemetry/instrumentation/rack/middlewares/event_handler.rb

```diff
@@ -109,7 +109,7 @@ def on_finish(request, response)
             span = OpenTelemetry::Instrumentation::Rack.current_span
             return unless span.recording?
 
-            add_response_attributes(span, response) if response
+            add_response_attributes(span, response) if response && !request.env.key?('rack.hijack?')
```

Comment:
> Is it likely that if the request is a rack.hijack, users wouldn't want their allowed response headers to be added as span attributes?

Yes, but I think they won't be available within the rack response we have access here because when the response is "hijacked", everything including response headers are deal with by another IO stream

Interesting bits in Rails are around here, I think:

https://github.com/rails/rails/blob/fe0fb9fdf04f638996e785d3e291f5fe8bc138a0/actioncable/lib/action_cable/connection/client_socket.rb#L59-L74

As you can see there, it returns a dummy Rack call result `[ -1, {}, [] ]` where `-1` is the status code, `{}` are the headers and `[]` is the body.